content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -541,13 +541,13 @@ Older versions of Kubernetes also supported the following in-tree PersistentVolu
 
 * `photonPersistentDisk` - Photon controller persistent disk.
   (**not available** starting v1.15)
-* [`scaleIO`](/docs/concepts/storage/volumes/#scaleio) - ScaleIO volume
+* `scaleIO` - ScaleIO volume.
   (**not available** starting v1.21)
-* [`flocker`](/docs/concepts/storage/volumes/#flocker) - Flocker storage
+* `flocker` - Flocker storage.
   (**not available** starting v1.25)
-* [`quobyte`](/docs/concepts/storage/volumes/#quobyte) - Quobyte volume
+* `quobyte` - Quobyte volume.
   (**not available** starting v1.25)
-* [`storageos`](/docs/concepts/storage/volumes/#storageos) - StorageOS volume
+* `storageos` - StorageOS volume.
   (**not available** starting v1.25)
```

Comment:
1. These 4 icons will now look like `photonPersistentDisk`.

2. I think we should also try making these 5 icons (`photonPersistentDisk`, `scaleIO`, `flocker`, `quobyte`, `storageos`) bold as well inside there code block as after removing hyperlink there text is looking very thin and light. Have a look at `photonPersistentIcon` at this page on K8s site.